### DISCUSSION TITLE
Internal Inertia Redirects outside store

### DISCUSSION THREAD

**Discussion by devlloedy** [inertiajs/inertia#352] [Help]
Hi I am wondering how to do an internal redirect ? I've already read the https://inertiajs.com/redirects , but it is possible to to make an internal redirect outside store contoller method ? thanks

**Comment by reinink** [comment]
Sorry, I'm not following. What do you mean by this?

> it is possible to to make an internal redirect outside store contoller method

Where, specifically, do you want to do that redirect?
